seqr/views/utils/terra_api_utils.py

```diff
@@ -110,12 +113,23 @@ def list_anvil_workspaces(user, fields=None):
 
 def user_get_workspace_access_level(user, workspace_namespace, workspace_name):
     path = "api/workspaces/{0}/{1}?fields=accessLevel".format(workspace_namespace, workspace_name)
+
+    cache_key = 'terra_req__' + hashlib.md5("{}__{}".format(user, path).encode('utf-8')).hexdigest()
+    r_text = safe_redis_get_json(cache_key)
```

Comment:
I'm a little confused b the variable name `r_text`, as that sounds like a raw text response, and this should be a parsed response dictionary right?

Reply:
I will rename it to `r`.